app/utils/tsinterval.js

```diff
@@ -0,0 +1,13 @@
+export default function tsinterval() {
+    let d1 = new Date(this.get('gte'));
```

Comment:
Also can you include the phrase "time series (ts)" somewhere in the comments so that it's clear what TS means?